data/allSolidTumorsOntology.json

```diff
@@ -0,0 +1,2480 @@
+{
```

Comment:
mostly all good, just noticed the filename. we're likely to add other diseases in the future so i would just call this diseases.json